api/src/handlers/helpers/structureInputData/common.ts

```diff
@@ -0,0 +1,51 @@
+import { InputFields } from "../getAllInputsFromForm";
+
+export interface CommonStructuredDataInput {
+  reference: string;
+  personalDetails: InputFields;
+  maritalStatus: InputFields;
+  consent: InputFields;
+  declaration: InputFields;
+  paymentDetails: InputFields;
+}
+
+export enum YesNoMap {
+  false = "No",
+  true = "Yes",
+}
+export function getCommonStructuredDataInput(
+  formData: InputFields
+): CommonStructuredDataInput {
+  const {
+    firstName,
+    middleName,
+    surname,
+    maritalStatus,
+    consent,
+    declaration,
+    paymentRef,
+    reference,
+  } = formData;
+
+  return {
+    reference: reference as string,
+    personalDetails: {
```

Comment:
I don't think the nesting is necessary. But I don't think this data structure is necessary either. At the moment there isn't a clear need to rename any component titles.